lib/src/kvs/ds.rs

```diff
@@ -237,24 +241,74 @@ impl Datastore {
 		}
 	}
 
-	// Adds entries to the KV store indicating membership information
-	pub async fn register_membership(&self) -> Result<(), Error> {
+	// -----
+	// Cluster helpers here
+	// -----
+	pub async fn cluster_init(&self) -> Result<(), Error> {
+		trace!("First register to prove cluster registration works");
+		let timestamp = self.register_membership().await?;
+		trace!("Healthcheck to perform garbage collection");
+		// node timeout should be configurable, just trying to get this to work
+		// let timeout = Duration::from_secs(1);
+		// let deadline = now - timeout;
+		let watermark = timestamp;
```

Comment:
I guess this is still a todo right? This won't delete old heartbeats

Reply:
Yes, absolutely. This PR is to rearrange things. The bigger PR (wip) is https://github.com/surrealdb/surrealdb/pull/2059